scripts/integration-tests/prometheus/prometheus-integration-test.sh

```diff
@@ -9,7 +9,7 @@ PARAM_TEST_BUILD="${TEST_BUILD:-true}"
 PARAM_TEST_VERIFY="${TEST_VERIFY:-true}"
 PARAM_TEST_TEARDOWN="${TEST_TEARDOWN:-true}"
 
-if [ $PARAM_TEST_BUILD != "true" ]; then
+if [ "$PARAM_TEST_BUILD" != "true" ]; then
```

Comment:
maybe rewire this test to just use your new m3_stack stuff?

Reply:
Also synced with @robskillington and want to keep them separate to make things easier, but also so the default configuration for the local stack can be a lot heavier